web/src/components/atomic/RenderMarkdown.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <span v-html="marked(props.content)" />
+</template>
+
+<script setup lang="ts">
+import { marked } from 'marked';
+
+const props = defineProps<{
+  content: string;
```

Comment:
about sanitizing ... if there is a script embeded as html coverd as markdown, is it sanitized?

Reply:
No, its not sanitized by default. I have not added sanitizing yet, as it's not really exposed to end-users and only handles code that we control. If you prefer to add sanitizing, I can add it of course.

Reply:
Works for me, will take care later today.